plugins/deps-core/src/main/java/tools/forma/deps/DependencyHandler.kt

```diff
@@ -36,6 +36,18 @@ fun DependencyHandler.addDependencyTo(
     configuration
 )
 
+// TODO: only using for platform, maybe make addPlatformDependencyTo?
+fun DependencyHandler.addDependencyTo(
```

Comment:
Whats real different of this method from same signature upper? Is't only in one parameter `Dependency` instead on `String`?